homeassistant/components/luftdaten/__init__.py

```diff
@@ -1,4 +1,8 @@
-"""Support for Luftdaten stations."""
+"""Support for Sensor.Community stations.
+
+Sensor.Community was previously called Luftdaten, hence the domain differs from
```

Comment:
Only tangentially related to this PR, but we probably need a way to rename config entry domains and migrate existing config entries so avoid user confusion.

`august` -> `yalexs`
`hunterdouglas_powerview` -> `powerview`
`flux_led` -> `magichome`
`luftdaten` -> `sensor.community`?

Reply:
Agreed, currently, ClimaCell is attempting such a thing, but... it's complex.